src/test/java/core/basesyntax/FruitDaoImplTest.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FruitDaoImplTest {
+    public static final String APPLE = "apple";
+    public static final String BANANA = "apple";
+    public static final int TEST_FRUIT_QUANTITY = 10;
+    private FruitDao fruitDao = new FruitDaoImpl();
+    private Fruit apple = new Fruit(APPLE, TEST_FRUIT_QUANTITY);
+
+    @BeforeEach
+    void setUp() {
+        fruitDao.add(apple);
+    }
+
+    @Test
+    void addFruit_notNull_success() {
+        assertNotNull(fruitDao.get(APPLE));
```

Comment:
Please check actual values, for example you expect Apple to be returned here, but you receive Banana and test still pass